src/main.rs

```diff
@@ -7,13 +7,14 @@ struct Envelope {
     results: Vec<Chip>
 }
 
+#[allow(non_snake_case)]
```

Comment:
This is interesting. Is it not possible to tell deserialiser to use snake_case?

Reply:
You are right, thank you.